x-pack/plugins/snapshot_restore/__jest__/client_integration/helpers/restore_snapshot.helpers.ts

```diff
@@ -6,23 +6,22 @@
  */
 import { act } from 'react-dom/test-utils';
 
-import { registerTestBed, TestBed, AsyncTestBedConfig } from '@kbn/test-jest-helpers';
+import { HttpSetup } from 'src/core/public';
+import { registerTestBed, TestBed } from '@kbn/test-jest-helpers';
 import { RestoreSnapshot } from '../../../public/application/sections/restore_snapshot';
 import { WithAppDependencies } from './setup_environment';
+import { REPOSITORY_NAME, SNAPSHOT_NAME } from '../helpers/constant';
 
-const testBedConfig: AsyncTestBedConfig = {
-  memoryRouter: {
-    initialEntries: ['/add_policy'],
-    componentRoutePath: '/add_policy',
-  },
-  doMountAsync: true,
```

Comment:
This was previously mounting the component on the wrong route, but since we had a wildcard api mock with sinon things were working anyway.